workflows/nanopore.nf

```diff
@@ -143,7 +143,7 @@ workflow NANOPORE {
             .map { dir ->
                 def count = 0
                 for (x in dir.listFiles()) {
-                    if (x.isFile() && x.toString().endsWith('.fastq')) {
+                    if (x.isFile() && x.toString().contains('.fastq')) {
```

Comment:
Is this really all it takes to get the pipeline working with `.gzip` files?! 😱 I do love it when a change this small takes you a long way!

Would you mind updating the `CHANGELOG` adding this entry please? 
https://github.com/nf-core/viralrecon/issues/218

And will eed to update the [`usage` docs](https://github.com/nf-core/viralrecon/blob/master/docs/usage.md#nanopore-input-format) to suggest that gzipped files are supported too.